site/en/docs/extensions/mv3/service_workers/index.md

```diff
@@ -96,9 +97,12 @@ chrome.bookmarks.onCreated.addListener(() => {
   // do something
 });
 ```
+{% endCompare %}
 
-Do not register listeners asynchronously, as they will not be properly triggered.
+If you put the same listener somewhere else, for examle, inside the `installed` listener, it may
+not be called when it is needed because it may not even be initialized yet.
```

Comment:
Typo + rewrite for clarity. Might be good to discuss this one as it's a bit weird to wrap your head around.

```suggestion
If you put the same listener somewhere else, for example, inside the `installed` listener, 
the event will be dropped because the nested listener will not be registered when the browser 
attempts to dispatch it.
```

Reply:
I like your rewrite, but for some reason, I can't commit it.

Reply:
Looks like the suggestion is outdated. I think you'll need to manually commit the change.